src/main/java/com/github/onsdigital/babbage/search/external/requests/suggest/SpellCheckRequest.java

```diff
@@ -0,0 +1,71 @@
+package com.github.onsdigital.babbage.search.external.requests.suggest;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.github.onsdigital.babbage.search.external.SearchEndpoints;
+import com.github.onsdigital.babbage.search.external.requests.base.AbstractSearchRequest;
+import com.github.onsdigital.babbage.search.external.requests.suggest.models.SpellingCorrection;
+import org.apache.http.client.utils.URIBuilder;
+import org.eclipse.jetty.client.api.ContentResponse;
+import org.eclipse.jetty.http.HttpScheme;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SpellCheckRequest extends AbstractSearchRequest<List<SpellingCorrection>> {
+
+    private final String searchTerm;
+
+    public SpellCheckRequest(String searchTerm) {
+        super(new TypeReference<List<SpellingCorrection>>() {});
+        this.searchTerm = searchTerm;
+    }
+
+    @Override
+    public URIBuilder targetUri() {
+        URIBuilder uriBuilder = new URIBuilder()
+                .setScheme(HttpScheme.HTTP.asString())
+                .setHost(HOST)
+                .setPath(SearchEndpoints.SPELLING.getEndpoint())
+                .addParameter("q", this.searchTerm);
```

Comment:
You could make this a field and populate it in the constructor then this method becomes a simpler getter - the benefit being you don't need to construct this object each time the method is called.